Source/Core/DolphinWX/Config/ConfigMain.cpp

```diff
@@ -78,7 +79,9 @@ void CConfigMain::CreateGUIControls()
 	wxPanel* const interface_pane = new InterfaceConfigPane(Notebook, ID_DISPLAYPAGE);
 	wxPanel* const audio_pane = new AudioConfigPane(Notebook, ID_AUDIOPAGE);
 #ifndef IS_PLAYBACK
-	wxPanel *const slippi_pane = new SlippiConfigPane(Notebook, ID_SLIPPIPAGE);
+	wxPanel* const slippi_pane = new SlippiConfigPane(Notebook, ID_SLIPPIPAGE);
```

Comment:
ifdef which class you use instead of creating a separate tab/page for playback settings. Keep the title of the tab as Slippi.